### ISSUE TITLE
Spec out a roadmap page for website / Figure out what to do with Iterative design strategy

### ISSUE THREAD

**Issue by krrisis** [commons-stack/iteration0#124]
## 🎉   Subtasks

---- These are Old ------

- [ ] Get a good understanding of how the future iterations will work 
- [ ] Draft text for iteration template 
- [ ] discuss [sitemap updates](https://miro.com/app/board/o9J_kvF68-0=/) with marko (FAQ & roadmap vs community)
- [ ] propose page for roadmap
- [ ] create page & push live

## 🤼‍♂️   Reviewer

@

**Comment by krrisis** [comment]
Being added as an FAQ question for now - @geleeroyale is looking into making the uploaded visual appear. Other visuals are now saved [over here](https://github.com/commons-stack/commons-stack-assets/tree/master/website/visuals), page can still be made in the future if relevant.

**Comment by krrisis** [comment]
Now [added to this FAQ](https://faq.commonsstack.org/#what-is-a-cultural-build) (thx for the fix @geleeroyale) - separate page definitely still a desired option I'd say.

**Comment by geleeroyale** [comment]
Since we built this on Gatsby we can create any new pages we want on top of the faq. The only thing to consider is the url of things. The site is reachable under faq.commonsstack.org, so any local routed would be i.e. faq.commonsstack.org/roadmap that we could then add a redirect to in Tilda (i.e. commons stack.org/roadmap

However it would still be more awesome to port the whole commonsstack.org site to Gatsby from Tilda as I have proposed previously. 

Those options should be heavily considered if there is a big push in functionality (i.e. the roadmap is interactive)
